saleor/cart/models.py

```diff
@@ -52,8 +52,15 @@ def canceled(self):
     def save(self):
         self.update(status=Cart.SAVED)
 
+    def with_products_data(self):
```

Comment:
Correct name would be `with_product_data` but I'd prefer to have it named after the purpose rather than its implementation detail. What would be a better name? `for_display`?